app/src/app.js

```diff
@@ -0,0 +1,47 @@
+const post = require('./model/post');
+const express = require('express');
+const connection = require('./model/connection');
+const app = express();
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
+
+app.post('/posts', async (req, res) => {
+    let title = req.body.title;
+    let slug = req.body.slug;
+    let content = req.body.content;
+    let addPost = new post.PostEntity(null, title, slug, content);
+
+    const error = [];
+    if (typeof title !== 'string') {
```

Comment:
Przerób to od razu żeby sprawdzać atrybuty encji np. `addPost.title`  zamiast tych zmiennych. Będzie czytelniej

Reply:
poprawione